pages/common/psql.md

```diff
@@ -2,22 +2,22 @@
 
 > PostgreSQL command-line client.
 
-- Connect to *database*. It connects to localhost using default port *5432* with default user:
+- Connect to database. It connects to localhost using default port 5432 with default user:
 
 `psql {{database}}`
 
-- Connect to *database* on given server *host* running on given *port* with *username* given, no password prompt:
+- Connect to database on given server host running on given port with given username, without a password prompt:
 
 `psql -h {{host}} -p {{port}} -U {{username}} {{database}}`
 
-- Connect to *database*, user will be prompted for password:
+- Connect to database; user will be prompted for password:
 
 `psql -h {{host}} -p {{port}} -U {{username}} -W {{database}}`
 
-- Run single *query* against the given *database*. Note: useful in shell scripts:
+- Execute a single command (sql or PostgreSQL-specific) against the given database. Note: useful in shell scripts:
 
 `psql -c '{{query}}' {{database}}`
 
-- Run several queries against the given *database*. Note: useful in shell scripts:
+- Execute commands from a file against the given database:
```

Comment:
same here. "against" may be idiomatic, but it's unnecessarily so. "on" works well and is less likely to confuse absolute beginners.